m1073496/who-would-win
# Feature/stats view

## What does this PR do (summary of changes)?
 - Creates the Stats component and provides a link to get there
 - Renders the correct character stats on character image click
 - Styles the Stats view - MVP
 - Adds a message for Stats view if no character has been selected
 - Adds back button functionality from Stats page
 - Cleans up typos in data

## How should it be tested?
 - User should be able to click on a character name, then click the image to get to that character's stat page
 - User should see valuable stats for that character
 - User should be able to navigate back to matchUp view 
 
## Future Iterations
- Render the correct character image on click / on stat page
- Set up API network requests to grab quotes
- Add outbound urls to take users to the sources of the power ranking data
- Fix bug that clears fighter selections on back button click
- Fix bug that clears fighter stats on refresh
